karbon/karbon_cluster_registration_service_test.go

```diff
@@ -118,6 +152,168 @@ func TestKarbonCreateClusterRegistration(t *testing.T) {
 	// TODO get task uuid status
 }
 
+func TestKarbonCreateClusterRegistrationAndSetInfo(t *testing.T) {
+	interceptor := khttpclient.NewInterceptor(http.DefaultTransport)
+	creds := testhelpers.CredentialsFromEnvironment(t)
+	nkeClient, err := NewKarbonAPIClient(creds, WithRoundTripper(interceptor))
+	require.NoError(t, err)
+
+	kctx := mock.NewContext(mock.Config{
+		Mode: keploy.MODE_TEST,
+		Name: t.Name(),
+	})
+
+	test_cluster_name := strings.ToLower("cluster1")
+	test_cluster_uuid := strings.ToLower("634F5852-CE1B-465C-A95A-9B8DFFBDDE42")
```

Comment:
just use lowercase to begin with :)

Reply:
sure but not needed. this was generated using uuidgen on mac